extensions/typescript-language-features/src/languageFeatures/completions.ts

```diff
@@ -760,6 +760,12 @@ class TypeScriptCompletionItemProvider implements vscode.CompletionItemProvider<
 			isIncomplete = !!response.body.isIncomplete || (response as any).metadata && (response as any).metadata.isIncomplete;
 			entries = response.body.entries;
 			metadata = response.metadata;
+			
+			if (response.body.optionalReplacementSpan) {
+				for (const entry of entries) {
+					entry.replacementSpan ??= response.body.optionalReplacementSpan;
```

Comment:
Sanity check: are there per-entry optional replacement spans, or is there only a single one to use for all of the entries?

Reply:
It's a single one to use for all of the entries, see https://github.com/microsoft/TypeScript/blob/a36d04fc63c83b6ec5a8099942b653a5caa29eb1/src/services/types.ts#L1382-L1384